pkg/amazon/backend/convert.go

```diff
@@ -142,17 +149,32 @@ func toLimits(service types.ServiceConfig) (string, string, error) {
 	}
 
 	for cpu, mem := range cpuToMem {
-		if v <= cpu*1024*1024 {
+		if v <= cpu*MiB {
 			for _, m := range mem {
-				if limits.MemoryBytes <= m*1024*1024 {
+				if limits.MemoryBytes <= m*MiB {
 					cpuLimit = strconv.FormatInt(cpu, 10)
 					memLimit = strconv.FormatInt(int64(m), 10)
 					return cpuLimit, memLimit, nil
 				}
 			}
 		}
 	}
-	return "", "", fmt.Errorf("unable to find cpu/mem for the required resources")
+	return "", "", fmt.Errorf("The resources requested are not supported by ECS/Fargate")
```

Comment:
nit: errors should always start with a lower case letter.

```suggestion
	return "", "", fmt.Errorf("the resources requested are not supported by ECS on Fargate")
```